docs/reference/commandline/run.md

```diff
@@ -134,89 +134,82 @@ previous changes intact using `docker start`. See `docker ps -a` to view a list
 of all containers.
 
 The `docker run` command can be used in combination with `docker commit` to
-[*change the command that a container runs*](commit.md). There is additional detailed information about `docker run` in the [Docker run reference](../run.md).
+[*change the command that a container runs*](commit.md).
+
+See the [Docker run reference](../run.md) for more details and examples about
+what you can do with the `docker run` command. A few examples are listed below.
 
 For information on connecting a container to a network, see the ["*Docker network overview*"](../../userguide/networking/index.md).
 
 ## Examples
 
-### Assign name and allocate pseudo-TTY (--name, -it)
-
-    $ docker run --name test -it debian
-    root@d6c0fe130dba:/# exit 13
-    $ echo $?
-    13
-    $ docker ps -a | grep test
-    d6c0fe130dba        debian:7            "/bin/bash"         26 seconds ago      Exited (13) 17 seconds ago                         test
-
-This example runs a container named `test` using the `debian:latest`
-image. The `-it` instructs Docker to allocate a pseudo-TTY connected to
-the container's stdin; creating an interactive `bash` shell in the container.
-In the example, the `bash` shell is quit by entering
-`exit 13`. This exit code is passed on to the caller of
-`docker run`, and is recorded in the `test` container's metadata.
+The `docker run` command is powerful and has many options which can augment or
+override your container's Dockerfile settings at runtime. See the
+[Docker run reference](../run.md) for more thorough examples using the
+`docker run` command. The following are only a few examples.
 
-### Capture container ID (--cidfile)
+### Simple examples
 
-    $ docker run --cidfile /tmp/docker_test.cid ubuntu echo "test"
+Run the `hello-world` container to test your Docker installation.
 
-This will create a container and print `test` to the console. The `cidfile`
-flag makes Docker attempt to create a new file and write the container ID to it.
-If the file exists already, Docker will return an error. Docker will close this
-file when `docker run` exits.
+```bash
+$ docker run hello-world
+```
 
-### Full container capabilities (--privileged)
+Run the `sh` command in an `alpine` container in interactive mode. Remove the
+container when it stops. To stop the container, type `exit`.
 
-    $ docker run -t -i --rm ubuntu bash
-    root@bc338942ef20:/# mount -t tmpfs none /mnt
-    mount: permission denied
+```bash
+docker run --rm -it alpine sh
+```
 
-This will *not* work, because by default, most potentially dangerous kernel
-capabilities are dropped; including `cap_sys_admin` (which is required to mount
-filesystems). However, the `--privileged` flag will allow it to run:
+Run a `nginx` container, mapping port 80 to port 8080 on the host machine. To
+access the web server, go to http://localhost:8080/.
```

Comment:
hm, always tricky, as `localhost` may not be where it's accessible, not sure if we should explain that here though